.svgrrc

```diff
@@ -0,0 +1,5 @@
+{
+  "replaceAttrValues": {
+    "#000": "{props.fill}"
```

Comment:
I assume this wouldn't cause any issues with existing SVG assets?

Reply:
This is a replacement color for bundles SVG files, atm the only bundled SVG files are the new icons. Also replacement color is only used if there it's passed through `fill` prop, otherwise it stays the same. So should not cause problems here.